### PR TITLE
Fix crash when version is too long

### PR DESCRIPTION
**Pull request by Zlopez**
Fixes https://github.com/release-monitoring/anitya/issues/674

Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: anitya/lib/utilities.py [hunk 1]
```diff
@@ -103,14 +103,16 @@ def check_project_release(project, session, test=False):
...
     p_versions = project.get_sorted_version_objects()
     old_version = project.latest_version or ''
+    version_column_len = models.ProjectVersion.version.property.columns[0].type.length
     for version in versions:
         if version not in p_versions:
-            project.versions_obj.append(
-                models.ProjectVersion(
-                    project_id=project.id,
-                    version=version.version
+            if len(version.version) < version_column_len:
```
Comment (cverna) [inline anitya/lib/utilities.py]: Should we log something when the version is too long ?

File: anitya/lib/utilities.py [hunk 2]
```diff
@@ -103,14 +103,16 @@ def check_project_release(project, session, test=False):
...
+                project.versions_obj.append(
+                    models.ProjectVersion(
+                        project_id=project.id,
+                        version=version.version
```
Comment (cverna) [inline anitya/lib/utilities.py]: The current column size is 50 char any reason for not making this bigger ? maybe 100 ?

File: anitya/lib/utilities.py (hunk 1 above)
Comment (Zlopez) [reply anitya/lib/utilities.py]: Yeah, we should. I will add a info level log message.

File: anitya/lib/utilities.py (hunk 2 above)
Comment (Zlopez) [reply anitya/lib/utilities.py]: It's enough for most projects, I only saw one project where this limit is not enough and this looks like mistake not a correct version. See https://github.com/jd/daiquiri/tags

Comment (Zlopez) [reply anitya/lib/utilities.py]: If this limit will be an issue in future we can update it, but right now it is enough.

Comment (cverna) [reply anitya/lib/utilities.py]: ok :+1:

**Pull request merged**